include/opw_kinematics/opw_parameters.h

```diff
@@ -14,10 +14,12 @@ struct Parameters
 
   T a1, a2, b, c1, c2, c3, c4;
   T offsets[6];
+  short int sign_corrections[6];
 
   Parameters()
     : a1{0}, a2{0}, b{0}, c1{0}, c2{0}, c3{0}, c4{4},
-      offsets{0, 0, 0, 0, 0, 0}
+      offsets{0, 0, 0, 0, 0, 0},
+      sign_corrections{1, 1, 1, 1, 1, 1}
```

Comment:
I have no real objection to using the signed short int here. I'm just curious why you didn't choose to use a signed char instead?

Reply:
I was not sure if `signed char` has the same behavior in calculations as an int. If it does, the smaller the better I suppose.